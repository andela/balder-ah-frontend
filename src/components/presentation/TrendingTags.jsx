import React from 'react';

const TrendingTags = () => (
  <div className="row">
    <div className="col s12">
      <ul className="tabs">
        <li className="tab col s2">
          <a href="#!">Life</a>
        </li>
        <li className="tab col s2">
          <a className="active" href="#!">
            Politics
          </a>
        </li>
        <li className="tab col s2">
          <a href="#!">Tech</a>
        </li>
        <li className="tab col s2">
          <a href="#!">Fashion</a>
        </li>
        <li className="tab col s2">
          <a href="#!">Music</a>
        </li>
        <li className="tab col s2">
          <a href="#!">Business</a>
        </li>
      </ul>
    </div>
  </div>
);

export default TrendingTags;
