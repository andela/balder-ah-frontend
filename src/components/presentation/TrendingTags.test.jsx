import React from 'react';
import { render, cleanup } from 'react-testing-library';
import 'jest-dom/extend-expect';
import TrendingTags from './TrendingTags';

afterEach(cleanup);

describe('<TrendingTags />', () => {
  test('should render correctly', () => {
    const { container } = render(<TrendingTags />);
    expect(container).toMatchSnapshot();
  });
});
