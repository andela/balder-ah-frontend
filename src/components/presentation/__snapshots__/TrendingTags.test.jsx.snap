// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TrendingTags /> should render correctly 1`] = `
<div>
  <div
    class="row"
  >
    <div
      class="col s12"
    >
      <ul
        class="tabs"
      >
        <li
          class="tab col s2"
        >
          <a
            href="#!"
          >
            Life
          </a>
        </li>
        <li
          class="tab col s2"
        >
          <a
            class="active"
            href="#!"
          >
            Politics
          </a>
        </li>
        <li
          class="tab col s2"
        >
          <a
            href="#!"
          >
            Tech
          </a>
        </li>
        <li
          class="tab col s2"
        >
          <a
            href="#!"
          >
            Fashion
          </a>
        </li>
        <li
          class="tab col s2"
        >
          <a
            href="#!"
          >
            Music
          </a>
        </li>
        <li
          class="tab col s2"
        >
          <a
            href="#!"
          >
            Business
          </a>
        </li>
      </ul>
    </div>
  </div>
</div>
`;
